plugins/BEdita/Core/src/Model/Action/ListRelatedObjectsAction.php

```diff
@@ -83,4 +85,23 @@ protected function buildQuery($primaryKey, array $data, Association $inverseAsso
 
         return $query;
     }
+
+    /**
+     * Prepare `joinData` entity on ObjectRelation associations.
+     * If relation is inverse switch between `priority`/`inv_priority`.
+     *
+     * @param \Cake\Datasource\EntityInterface $joinData Join data entity.
+     * @return void
+     */
+    protected function prepareJoinEntity(EntityInterface $joinData): void
+    {
+        if (!$joinData instanceof ObjectRelation || $this->Association->getForeignKey() !== 'right_id') {
```

Comment:
Maybe we could use `$this->Association instanceof RelatedTo` and add a method as `RelatedTo::isInverse()` that return `$this->Association->getForeignKey() === 'right_id'`